dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -107,17 +107,51 @@ const LiveGrid = ({ children }: { children: React.ReactNode }) => (
 			${from.desktop} {
 				margin-left: 320px;
 			}
-
 			@supports (display: grid) {
 				display: grid;
 				width: 100%;
 				margin-left: 0;
-
 				grid-column-gap: 10px;
+				${until.desktop} {
+					grid-template-columns: 1fr; /* Main content */
+					grid-template-areas:
+						'standfirst'
+						'lines'
+						'meta';
+				}
+				${from.desktop} {
+					grid-template-columns: 309px 1px 1fr;
```

Comment:
@DavidLawes earlier work to remove `border` and change left col width will need to be respected on any merge here

Reply:
Yea I can see there has been changes in `LiveGrid` & `HeadlineGrid`. @oliverlloyd  & @DavidLawes Could I pair with one of you to make sure what needs to change in `StandFirstGrid`?

Reply:
I tried to do what David had done. You can see the changes in this commit https://github.com/guardian/dotcom-rendering/pull/3655/commits/af28f250e9f932b6c7a30acb96cf8f168319a72b
Hope it's all OK

Reply:
That looks about right!